ui/src/components/settings/sub_pages/profile/mod.rs

```diff
@@ -538,3 +564,22 @@ fn get_input_options(validation_options: Validation) -> Options {
         ..Options::default()
     }
 }
+
+fn get_status_option<'a>(cx: Scope<'a>, status: &IdentityStatus) -> (String, Element<'a>) {
+    let indicator = Status::from(*status);
+    (
+        serde_json::to_string::<IdentityStatus>(status).unwrap_or_default(),
```

Comment:
Since `IdentityStatus` implements `Display` (indirectly), you could use `status.to_string()` instead.

Reply:
that way though you cant get it back from the string. as to_string and deserialization provide different strings

Reply:
when selecting it parses it back to `IdentityStatus` 
at https://github.com/Satellite-im/Uplink/pull/1571/files/68489ff7fb34364820dc0db7d3ffbee6564be160#diff-993f168fccb56d7662f6c89e41a0ff85392c38375ac2e8e9d1d230bcc580ddb9R449